src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -7,39 +7,37 @@ import React, {
   useState,
 } from 'react';
 import PropTypes from 'prop-types';
+import useFetch from '@bloodyaugust/use-fetch';
 import Select from '../select/select.jsx';
 import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+const defaultValue = [-1];
```

Comment:
I'm see there is other code referencing `-1`. I feel like using the default value of `[]` should be better. What's the reasoning for using `[-1]`?

Reply:
For posterity, this is used to capture the empty case. It is useful to have a fixed sized array for spreading into the `useEffect`'s dependencies parameter.